projects/angular/src/forms/common/wrapped-control.ts

```diff
@@ -103,13 +103,6 @@ export class WrappedFormControl<W extends DynamicWrapper> implements OnInit, OnD
   @HostListener('blur')
   triggerValidation() {
     if (this.ifControlStateService) {
-      /**
-       * For some reason the <input type="number" /> on blur ngControl doesn't set the control to 'touched'
-       * This one is a workaround to provide the control to be 'touched' on blur and fix #4480.
-       */
-      if (this.ngControl && !this.ngControl.touched) {
-        this.markAsTouched();
-      }
       this.ifControlStateService.triggerStatusChange();
```

Comment:
I wanted to remove this one too, because it forces `verifyOn: blur` like behavior and overrides the built-in angular forms strategies.
It is a big breaking change though and it will not be welcomed by many users that seem to prefer this type of default.
We do not have any complaint about this, so I have stashed the bigger fix for now.